.github/workflows/docbuild.yml

```diff
@@ -19,4 +22,4 @@ jobs:
       - run: pip install mkdocs-material
       - run: npm install
       - run: npx typedoc --out docs/api lib/index.ts
-      - run: mkdocs gh-deploy
+      - run: mkdocs gh-deploy --force
```

Comment:
what is up with --force? I never had to it when deploying from my box.

Reply:
Its not required, i added it to try out to see if any issues. Removed it now.

Reply:
It works well only if you have `--force` with GH Actions. See it is tested with this PR and works fine.